indigo_api/migrations/0132_migrate_ids_eIds.py

```diff
@@ -0,0 +1,41 @@
+# Generated by Django 2.2.12 on 2020-05-16 12:33
+
+import json
+from reversion.models import Version
+
+from django.contrib.contenttypes.models import ContentType
+from django.db import migrations
+
+from indigo_api.data_migrations import AKNeId
+
+
+def forward(apps, schema_editor):
+    db_alias = schema_editor.connection.alias
+    Document = apps.get_model("indigo_api", "Document")
+    ct_doc = ContentType.objects.get_for_model(Document)
+    migration = AKNeId()
+
+    for document in Document.objects.using(db_alias).all():
+        xml = migration.update_xml(document.document_xml)
+        if xml != document.document_xml:
+            document.document_xml = xml
+            document.save()
+
+        # Update historical Document versions
+        for version in Version.objects.filter(content_type=ct_doc.pk)\
+                .filter(object_id=document.pk).using(db_alias).all():
+            data = json.loads(version.serialized_data)
+            data[0]['fields']['document_xml'] = migration.update_xml(data[0]['fields']['document_xml'])
+            version.serialized_data = json.dumps(data)
+            version.save()
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('indigo_api', '0131_migrate_namespaces'),
+    ]
+
+    operations = [
+        migrations.RunPython(forward, migrations.RunPython.noop),
```

Comment:
I think remove the backwards migration, to indicate to Django that it can't go backwards.